env.js

```diff
@@ -1,3 +1,14 @@
 import dotenv from "dotenv";
 
-dotenv.config({});
+let environment = "dev";
```

Comment:
It works for me with this code:
```js
import dotenv from "dotenv";

dotenv.config({ path: process.env.ENV ? `./.env.${process.env.ENV}` : "./.env" });
```

And in the package.json file:
```json
"scripts": {
    "start": "nodemon --experimental-modules ./bin/www.js",
    "test": "ENV=test mocha '__tests__/*.js'"
  },
```
I added ENV=test before mocha. Your solution works well too but it's better if there is less code :)